### ISSUE TITLE
Error during 2.4 pre-test with A3A_fnc_addTimeForIdle

### ISSUE THREAD

**Issue by Bob-Murphy** [official-antistasi-community/A3-Antistasi#1746]
```
arma3server_x64_2021-02-25_18-34-54 - Test 1 - Server

2021/02/26, 12:12:31 "63424.7: [Antistasi] | INFO | fn_scheduler | Scheduled function: A3A_fnc_convoy, Function params: [""resource"",""""]"
2021/02/26, 12:12:31 Server: Object 2:59508 not found (message Type_124)
2021/02/26, 12:12:31 Error in expression < + _time];
if (_isMarker) then {server setVariable [_siteX,dateToNumber _dateAr>
2021/02/26, 12:12:31   Error position: <setVariable [_siteX,dateToNumber _dateAr>
2021/02/26, 12:12:31   Error Reserved variable in expression
2021/02/26, 12:12:31 File functions\Base\fn_addTimeForIdle.sqf [A3A_fnc_addTimeForIdle]..., line 23
```

**Comment by jaj22** [comment]
A3A_fnc_missionRequest doesn't safety-check the return value from findBasesForConvoy, so it's possible for A3A_fnc_convoy to be called with a garbage source base if there are none available. This will fail.

**Comment by MeltedPixel** [comment]
Issue resolved previously before 3.0
